### PR TITLE
NID: Add __stack_chk_guard

### PR DESCRIPTION
**Pull request by EXtremeExploit**

### CODE REVIEW

**Comment by Zangetsu38** [comment]
missing function register

**Comment by EXtremeExploit** [comment]
register?

**Comment by Zangetsu38** [comment]
yes register function stub

**Comment by EXtremeExploit** [comment]
what?

**Comment by Macdu** [comment]
@Zangetsu38 The stub implementation is already in the code (even before this PR), it's only the NID and bridge part that was missing.

**Review by bookmist — APPROVED** [review]

File: vita3k/modules/SceLibKernel/SceLibKernel.cpp [hunk 1]
```diff
@@ -1696,6 +1696,7 @@ EXPORT(int, sceSblGcAuthMgrSclkSetData2) {
 BRIDGE_IMPL(__sce_aeabi_idiv0)
 BRIDGE_IMPL(__sce_aeabi_ldiv0)
 BRIDGE_IMPL(__stack_chk_fail)
+BRIDGE_IMPL(__stack_chk_guard)
```
Comment (Zangetsu38) [inline vita3k/modules/SceLibKernel/SceLibKernel.cpp:1699]: register function inside this file, for now only set nid no have sense

**Review by Zangetsu38 — CHANGES_REQUESTED** [review]

**Comment by Zangetsu38** [comment]
> @Zangetsu38 The stub implementation is already in the code (even before this PR), it's only the NID and bridge part that was missing.

ah i see yes, very strange that

**Pull request merged**